simple.css

```diff
@@ -180,6 +180,7 @@ a:hover {
 
 button,
 [role="button"],
+a[role="button"],
```

Comment:
we're already selecting for `[role="button"]`, do we need a duplicate selector for `a`?

Reply:
Good point!